Formula/e/envelope.rb

```diff
@@ -0,0 +1,33 @@
+class Envelope < Formula
+  desc "Environment variables CLI tool"
+  homepage "https://github.com/mattrighetti/envelope"
+  url "https://github.com/mattrighetti/envelope/archive/refs/tags/0.3.11.tar.gz"
+  sha256 "1a378564b07e041fbf3212655e8c6442f8973080cf0698886764ce38982661cc"
+  license :public_domain
+  head "https://github.com/mattrighetti/envelope.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "pandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    args = %w[
+      --standalone
+      --to=man
+    ]
+    system "pandoc", *args, "man/envelope.1.md", "-o", "envelope.1"
+    man1.install "envelope.1"
+  end
+
+  test do
+    envelope_output = shell_output("#{bin}/envelope --version").strip
+    assert_equal "envelope #{version}", envelope_output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I've added more tests, thanks!